doc/52_Tips_and_Tricks.md

```diff
@@ -10,7 +10,7 @@
 
 ## Command Palette
 
-The _command palette_ is the center for a keyboard-centric interaction, as well as for looking up
+The _command palette_ is the core of a keyboard-centric interaction, and can be also useful for looking up
```

Comment:
is `at the core of ...`

I liked the `as well as` better - it's shorter and conveys the same information

Reply:
fixing to _at the core of ..._